components/JobModal.js

```diff
@@ -13,6 +14,9 @@ export default class JobModal extends React.Component {
       client: this.props.email,
       title: "",
       categories: [],
+      coordinates: { lat: null, lng: null },
+      lat: 0,
```

Comment:
Please remove these two pieces of state... They are redundant, and I expect more from you.